src/components/atoms/DatetimeInput/DatetimeInput.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <!-- datetimeinput -->
+  <div class="">
+    <input type="text" id="datetime-input"
+      class="h-[42px] w-[259px] text-gray-600 uppercase text-sm rounded border-gray-300" placeholder="MM/YY"
+      v-model="date" @input="formatDate" maxlength="5" />
+  </div>
+  <!-- datetimeinput end -->
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+
+const date = ref('');
+const formatDate = () => {
+  let formattedDate = date.value.replace(/[^\d]/g, '').slice(0, 4);
+  if (formattedDate.length === 4) {
+    formattedDate = formattedDate.slice(0, 2) + '/' + formattedDate.slice(2);
+  }
+  date.value = formattedDate;
```

Comment:
chỗ format anh bảo dùng thư viện mà nhỉ
```suggestion
import moment from 'moment'
const formatDate = moment(date).format(DateFormat.YYYYMM)
```
Khai báo enum trong file ```shared/enum/format.ts```
```
export enum DateFormat {...}
```